ui/components/PatternConfigurator/PatternConfigurator.js

```diff
@@ -0,0 +1,358 @@
+import React, { useState, useContext, useEffect } from "react";
+import {
+  AppBar,
+  FormControl,
+  IconButton,
+  MenuItem,
+  TextField,
+  Toolbar,
+  Tooltip,
+  Autocomplete,
+  Select,
+} from "@mui/material";
+import { styled } from "@mui/system";
+import DeleteIcon from "@mui/icons-material/Delete";
+import FileCopyIcon from "@mui/icons-material/FileCopy";
+import ListAltIcon from "@mui/icons-material/ListAlt";
+import SaveIcon from "@mui/icons-material/Save";
+import NameToIcon from "./NameToIcon";
+import {getMeshProperties} from "../../utils/nameMapper"
+import { isEmptyObj } from "../../utils/utils";
+import { getHumanReadablePatternServiceName, getPatternServiceName } from "./helpers";
+
+const PattrenBar = styled(AppBar)(() => ({
+  marginBottom: "16px",
+  backgroundColor: "#fff",
+  borderRadius: "8px"
+}));
```

Comment:
Use `theme.spacing` instead of `px` and `theme.palette` to define the white color